## ***SI 311 Homework #10: Identifying Sue Bird's Value as a Player***

Data from: https://herhoopstats.substack.com/p/updated-2020-wnba-rosters-and-salary and https://www.basketball-reference.com/wnba/years/2021_advanced.html

Formulas from SI 311, University of Michigan Fall '22

*Code Written by Brandon Huggard*

https://www.linkedin.com/in/brandon-huggard-5957a0192/

In [46]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

In [47]:
sals = pd.read_csv('wnba21.csv')

In [73]:
bird_sal = 215000

In [49]:
sals.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 12 entries, 0 to 11
Data columns (total 5 columns):
 #   Column               Non-Null Count  Dtype  
---  ------               --------------  -----  
 0   TEAM                 12 non-null     object 
 1   TOTAL SALARIES       12 non-null     int64  
 2   TOTAL PLAYERS        12 non-null     float64
 3   CAP ROOM             12 non-null     object 
 4   GUARANTEED SALARIES  12 non-null     object 
dtypes: float64(1), int64(1), object(3)
memory usage: 608.0+ bytes


In [50]:
sals

TEAM  TOTAL SALARIES  TOTAL PLAYERS    CAP ROOM  \
0        Atlanta Dream         1339000           13.0        0.00   
1          Chicago Sky         1338778           11.0      222.00   
2      Connecticut Sun         1352115           11.0  -13,115.00   
3         Dallas Wings         1243559           12.0   95,441.00   
4        Indiana Fever         1277179           12.0   61,821.00   
5       Las Vegas Aces         1345922           11.0   -6,922.00   
6   Los Angeles Sparks         1338518           11.0      482.00   
7       Minnesota Lynx         1346277           12.0   -7,277.00   
8     New York Liberty         1338063           12.0      937.00   
9      Phoenix Mercury         1353144           11.0  -14,144.00   
10       Seattle Storm         1336367           13.0    2,633.00   
11  Washington Mystics         1278114           12.0   60,886.00   

   GUARANTEED SALARIES  
0           798,700.00  
1           953,375.00  
2         1,053,496.00  
3           716,381.00  
4           645,991.00  
5           969,110.00  
6           882,643.00  
7           979,210.00  
8           855,907.00  
9           874,513.00  
10          725,569.00  
11          805,556.00

In [61]:
total_cash = sals['TOTAL SALARIES'].sum()

In [62]:
total_cash

15887036

Each team played 32 games this season- https://en.wikipedia.org/wiki/2021_WNBA_season#:~:text=Due%20to%20the%20COVID%2D19,game%20series%20to%20reduce%20travel.

12 teams

In [63]:
total_wins = 32 * 6
total_wins

192

In [65]:
win_val = total_cash / total_wins
win_val

82744.97916666667

Note: Sue Bird played 30 games in 2021

In [68]:
bird_ws = 3

In [70]:
bird_value = win_val * bird_ws

In [71]:
bird_value

248234.9375

In [76]:
diff = bird_value - bird_sal
diff

33234.9375

Sue Bird generated $33234.94 *MORE* in win vaue than what she was paid in the 2021 season.